apps/condo/domains/notification/utils/serverSchema/helpers.js

```diff
@@ -0,0 +1,98 @@
+const get = require('lodash/get')
+
+const {
+    MESSAGE_DELIVERY_OPTIONS,
+    DEFAULT_MESSAGE_DELIVERY_OPTIONS,
+} = require('@condo/domains/notification/constants/constants')
+const { NotificationUserSetting } = require('@condo/domains/notification/utils/serverSchema/index')
+
+// Settings priorities
+// The highest number is the most prioritized value
+const ALL_MESSAGES_TYPES = 10
```

Comment:
ALL_MESSAGE_TYPES_PRIORITY
ALL_TRANSPORT_TYPES_PRIORITY
PARTICULAR_????

Reply:
The priority value for a particular message type and transport. This is the most prioritized case.